.npmignore

```diff
@@ -1,29 +1,27 @@
 Gruntfile.js
```

Comment:
Bin kein Fan von `.npmignore`. Lieber explizit auflisten, was ins package soll. Siehe auch https://github.com/ioBroker/create-adapter/pull/816/files#diff-d6b0827e37f1786fbdbcb9f31db351a7f1067d39e588bddff5ec0971475535c9

Reply:
Ok for me